src/sentry/monitors/endpoints/organization_monitor_details.py

```diff
@@ -139,12 +140,12 @@ def delete(self, request: Request, organization, project, monitor) -> Response:
         """
         Delete a monitor or monitor environment.
         """
-        environment_name = request.query_params.get("environment")
+        environment_names = request.query_params.getlist("environment")
```

Comment:
should we also change the operation_id text above to "Delete a monitor or monitor **environments**"?

Reply:
ah. yeah. I would have enforced singular but didn't want to write a new OpenAPI spec for the decorator